eZ/Publish/API/Repository/Tests/SearchServiceFieldFiltersSolrTest.php

```diff
@@ -623,6 +621,7 @@ public function testFullTextFilterLanguageEmpty()
     /**
      * Test for the findContent() method.
      *
+     * @group ttt
```

Comment:
Yeah, I usually use aaa and zzz, but ran out of letters :)
Since this file is removed in https://github.com/ezsystems/ezpublish-kernel/pull/1322, if you agree I would skip this to avoid conflicts.